client/src/components/pages/QuizSend.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+import {Nav, Tab, Row, Col, Container, Form, Button} from 'react-bootstrap';
+import axios from "axios";
+import { useNavigate } from "react-router-dom";
+
+
+function QuizSend(){
+    //collect all email addresses as a single string
+    const [eAddress, setEmail] = useState({
+        emailText: ""
+      });
+
+    const[eAddresses, setState] = useState([])
+    const navigate = useNavigate();
+
+    function getEmail(event){
+        const {name, value} = event.target;
+        setEmail({
+          [name]: value,
+        });
+    }
+ 
+
+    function onSubmit(event){
+        event.preventDefault()
+        let emailList = eAddress.emailText.split(","); //email into array
+        console.log(emailList);
+
+        let emailSubject = "You are invited to take a quiz";    //set subject and context
+        let emailContext = "Test email";                        //we can modify the text later
+
+        if(emailList[0] == ''){
+            alert("Please enter at least one email address");
+        } else{
+            for(let i = 0; i < emailList.length; i++){              //axios to send data to backend to send out emails
+                axios({
+                    method: "POST",
+                    url:"http://localhost:7000/send",               //we might need to change url for actual url for backend
```

Comment:
yes, I agree. Probably good idea to have email validations as well as done for register and edit profile

Reply:
Can you guys help me to validate email input?